.github/workflows/build-citus-community-nightlies.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
 
       - name: Clone tools branch
-        run: git clone -b v0.8.24 --depth=1  https://github.com/citusdata/tools.git tools
```

Comment:
Why do you need to update tools version on ontouched workflows? I do not like how this looks.

If you always wish to have these versions up to date consider implementing https://github.com/citusdata/packaging/issues/948

Reply:
I added comments in the issue #948 . I'm changing versions one by one after executing all tests for that pipeline